CommonCoreLegacy/src/test/java/com/skedgo/tripkit/routing/TripSegmentTest2.java

```diff
@@ -152,6 +152,7 @@
         .isEqualTo("To City Circle. Platform: Platform 2");
   }
 
+  /*
   @Test public void shouldRemovePlatformTemplateIfNoPlatformAvailable() {
```

Comment:
This test would be good to keep, but it's not super urgent. Generally these "templates" should never be visible in the UI, and if there's no value to replace them with they should be removed.